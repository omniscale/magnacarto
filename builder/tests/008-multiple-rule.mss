#class1,
#class2 {
}

#class1[foo=12],
#class2[bar=11]
{
    line-width: 99
}

