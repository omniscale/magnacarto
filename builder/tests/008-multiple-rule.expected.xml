<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="class1" filter-mode="first">
    <Rule>
      <Filter>([foo] = 12)</Filter>
      <LineSymbolizer stroke-width="99"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="class2" filter-mode="first">
    <Rule>
      <Filter>([bar] = 11)</Filter>
      <LineSymbolizer stroke-width="99"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="class1" srs="" status="off">
    <StyleName>class1</StyleName>
  </Layer>
  <Layer name="class2" srs="" status="off">
    <StyleName>class2</StyleName>
  </Layer>
</Map>